css/style.css

```diff
@@ -16,4 +12,259 @@ body {
  * - When using Flexbox, remember the items you want to move around need to be inside a parent container set to 'display: flex'
  */
 
+/* General styles */
+
+:root {
+    
+    --grey: #616668;
+}
+
+html, body {
+    font-family: 'Roboto', sans-serif;
+    -webkit-font-smoothing: antialiased;
+    margin: 0;
+    padding: 0;
+    box-sizing: border-box;
+    width: 100%;    
+    font-family: Roboto, sans-serif; 
```

Comment:
The font in your page is showing up as `sans-serif` for me -- I think this may be due to the duplicated `font-family` property in this section. Notice how this one (which most likely overwrites the top one, as css is ranked from bottom to top) does not put `Roboto` in quotes -- maybe that's why?